src/structs.md

```diff
@@ -17,3 +17,17 @@ fn main() {
     println!("{} is {} years old", peter.name, peter.age);
 }
 ```
+
+<details>
+
+* The top of the program is how we define a struct, the main function creates a struct, and then at the bottom we access the fields.
+* Structs work similar as they do in other languages, but note there is also no typedef keyword or inheritance between structs. Also note there are no functions. The implementation is separate.
+* We can access the structs fields with the dot notation 
+* Using short-hand syntax, we can initiate the fields with similar-named variables.  [define age variable on it’s own line] 
+* There are multiple types of syntax for structs in Rust
+    * Unit_ like struct, can Define with “struct (name);” and no need for curly brackets and parentheses.
```

Comment:
Please mark all code with backticks. Your curly quotes are nice from a typographical standpoint, but it's the wrong markup here :smile:

Reply:
There are some more typos here: `Unit_ like` should probably be `Unit-like` and the capital `D` in `Define` is out of place.